packages/twenty-server/src/modules/workflow/common/types/workflow-step.type.ts

```diff
@@ -12,7 +22,22 @@ type BaseWorkflowStep = {
 
 export type WorkflowCodeStep = BaseWorkflowStep & {
   type: WorkflowStepType.CODE_ACTION;
-  settings: WorkflowCodeSettings;
+  settings: BaseWorkflowSettings & {
```

Comment:
are these the settings of the step, or the settings of the workflow?

Reply:
step